nodes/src/nodes/llm_gmi_cloud/gmi_cloud.py

```
@@ -0,0 +1,88 @@
+            temperature=0,
+            max_tokens=self._modelOutputTokens,
+        )
+
+        bag['chat'] = self
+
+    def is_retryable_error(self, error):
+        """Determine if the error is retryable."""
+        if isinstance(error, RateLimitError):
+            return True
+        elif isinstance(error, APIConnectionError):
+            return True
```

Ruff (0.15.7)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)